src/components/RequestInput.vue

```diff
@@ -1,5 +1,5 @@
 <script setup>
-import { ref } from 'vue';
+import { ref, toRaw } from 'vue';
```

Comment:
Could you explain why we need `toRaw` here? Does the proxied object not work with JSON.stringify?